### DISCUSSION TITLE
Setting blend mode kills performance

### DISCUSSION THREAD

**Discussion by HorstBaerbel** [vcync/modV#871] [Q&A] [answered]
I'm on Fedora 38 (Gnome + Wayland), Ryzen 5700X, Nvidia GTX 3060 and use modV 3.28.0. My presets run fine, until I set any blend mode of any enabled module to anything but normal. Then performance tanks (like 10-100x) and modV becomes almost unusable. Attached my preset. Doing anything wrong?
[2023-05.zip](https://github.com/vcync/modV/files/11808494/2023-05.zip)

**Comment by 2xAA** [comment]
Hm, strange. Nothing wrong here. Running this on macOS Ventura 13.3 on an M1 Max, 64GB.

However, blend modes shouldn't cause anything to tank. This isn't custom functionality, just canvas' compositeOperation (https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation).

Maybe you've found a Chrome bug?

We've just released modV 3.29.0 which updates Electron. Could you try again with the new release please?

**Reply by 2xAA** [reply]
Oh, er, scratch that - there's an issue with the build for 3.29.0. I'll reply once it's resolved, then you can test.

**Reply by 2xAA** [reply, marked as answer]
Okay - 3.29.1 released :)

**Reply by HorstBaerbel** [reply]
Seems to be gone on 3.29.1 \o/

**Reply by 2xAA** [reply]
Glad to hear it :)
